sc-24sp/prototypes/prototype-12/README.md

```diff
@@ -0,0 +1,94 @@
+So as per the instructions, I copied most of the work from prototype-0.
+When I try to run the cargo project, I recieve these errors:
+
+```
+
+error[E0433]: failed to resolve: use of undeclared crate or module `prototype_0`
```

Comment:
Thanks for reporting this. This is because the cargo project in the prototype_12 directory is called `prototype-12`.

Some of the imports or `use` statements have hard-coded cargo project names, and it would be better to find a relative path for those with the `crate::` top-level namespace or `super::` relative namespaces. Try fixing this in your `prototype-12` folder.